lib/screen/boarding_screen.dart

```diff
@@ -0,0 +1,136 @@
+import 'package:flutter/material.dart';
+import 'package:sochem/screen/slide.dart';
+
+class BoardingPage extends StatefulWidget {
+  @override
+  _BoardingScreenState createState() => _BoardingScreenState();
+}
+
+class _BoardingScreenState extends State<BoardingPage> {
+  int _currentPage = 0;
+  List<Slide> _slides = [];
+  PageController _pageController = PageController();
```

Comment:
This Slide class does not exist in the code you have pushed, I assume you forgot to add the slide.dart into the commit you pushed.